__init__.py

```diff
@@ -146,8 +146,7 @@
 VERSION = '3.22.2.2'
 
 from collections import defaultdict
-from arelle import PythonUtil  # define 2.x or 3.x string types
-PythonUtil.noop(0)  # Get rid of warning on PythonUtil import
+from arelle import PythonUtil

```

Comment:
This import of PythonUtil is used elsewhere below. The noop ref to avoid warnings on an unused import was already no longer required.